src/openzeppelin/token/erc721/interface.cairo

```diff
@@ -0,0 +1,101 @@
+use openzeppelin::utils::serde::SpanSerde;
+use starknet::ContractAddress;
+use array::SpanTrait;
+
+const IERC721_ID: u32 = 0x80ac58cd_u32;
+const IERC721_METADATA_ID: u32 = 0x5b5e139f_u32;
+const IERC721_RECEIVER_ID: u32 = 0x150b7a02_u32;
+
+#[abi]
+trait IERC721ABI {
```

Comment:
```suggestion
trait ERC721ABI {
```

I used AccountABI and not IAccountABI to further separate the concept of the interface and the ABI in the pattern. From my thinking, this trait refers to the ABI for Dispatcher usage, not to the interface.